_config.php

```diff
@@ -1,3 +0,0 @@
-<?php
-
-Object::add_extension("ModelAdmin", "VersionedModelAdmin");
```

Comment:
Shouldn't all Silverstripe files have a newline at the end? config.php and config.yml don't.